src/main.rs

```diff
@@ -136,11 +132,7 @@ fn pass_to_pnpm(args: &[OsString]) -> Result<(), MainError> {
 }
 
 fn pass_to_sub(command: String) -> Result<(), MainError> {
-    let mut path_env = OsString::from("node_modules/.bin");
-    if let Some(path) = env::var_os("PATH") {
-        path_env.push(":");
-        path_env.push(path);
-    }
+    let path_env = get_prepended_path_env("node_modules/.bin");
```

Comment:
It's `node_modules\\.bin` on Windows.

```suggestion
    let path_env = Path::new("node_modules")
        .join(".bin")
        .pipe(get_prepended_path_env);
```